doc/man3/ERR_print_errors.pod

```diff
@@ -16,12 +16,15 @@ ERR_print_errors, ERR_print_errors_fp, ERR_print_errors_cb
 
 =head1 DESCRIPTION
 
-ERR_print_errors() is a convenience function that prints the error
+ERR_print_errors() prints the contents of the current
+error stack to the BIO B<bp>, and empties the error queue.
 strings for all errors that OpenSSL has recorded to B<bp>, thus
 emptying the error queue.
```

Comment:
These two leftover lines should be deleted.

Reply:
fixup commit pushed.